### PR TITLE
Added readme file

### PR DESCRIPTION
**Pull request by PasinduBuddhika**

### CODE REVIEW

**Comment by BuddhikaPasindu** [comment]
Add more lines of text describing what the program does in your readme file

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Just adding some text to the readme file
```
Comment (BuddhikaPasindu) [inline README.md]: This line of text is not suitable

File: README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+This is the readme file for the word order python project
```
Comment (BuddhikaPasindu) [inline README.md:2]: Add little more details to this line

**Comment by PasinduBuddhika** [comment]
All the comments have been resolved, please take another look.

**Pull request merged**